package/kernel/linux/modules/netdevices.mk

```diff
@@ -1141,3 +1141,23 @@ define KernelPackage/mlx5-core/description
 endef
 
 $(eval $(call KernelPackage,mlx5-core))
+
+
+define KernelPackage/sfp
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=SFP cage support
+  DEPENDS:=+kmod-i2c-core +kmod-hwmon-core
+  KCONFIG:= \
+  	CONFIG_SFP=m \
```

Comment:
can you replace these spaces with a tab?

Reply:
Well spotted, Fixed
Thanks!